src/components/Block/Block.test.tsx

```diff
@@ -1,27 +1,21 @@
 /*! Copyright (c) Microsoft Corporation. All rights reserved. Licensed under the MIT license. */
 import * as React from 'react';
-import { createRenderer as createShallowRenderer } from 'react-test-renderer/shallow';
+import { create as createRenderer, ReactTestRendererJSON } from 'react-test-renderer';
 import { renderIntoDocument, findRenderedDOMComponentWithClass } from 'react-dom/test-utils';
 import Block, { GutterSize, TextColor, TextSize } from '.';
+import Clickable from '../Clickable';
 
-const shallowWithContext = (jsx: JSX.Element) => {
-  const renderer = createShallowRenderer();
-  renderer.render(jsx);
-  return renderer.getRenderOutput();
-};
-
-const renderWithoutContext = (jsx: JSX.Element) => {
-  const renderedText = renderIntoDocument(jsx) as Block;
-  return findRenderedDOMComponentWithClass(renderedText, 'y-block');
+const render = (jsx: JSX.Element) => {
+  return createRenderer(jsx).toJSON();
 };
 
 describe('<Block />', () => {
-  let component: Element;
-  let componentWithContext: React.ReactElement<Block>;
+  let component: ReactTestRendererJSON | null;
+  let componentWithContext: ReactTestRendererJSON | null;
 
   describe('with default options', () => {
     beforeEach(() => {
-      componentWithContext = shallowWithContext(<Block>block content</Block>);
+      componentWithContext = render(<Block>block content</Block>);
```

Comment:
Previously this would capture the context provider, which was good for ensuring that the correct value was set.  With the addition of the clickable context consumer tag, I'm having trouble finding a away to render it in a way that still captures it.